eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -62,16 +62,16 @@ jobs:
           - (Centos.8.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-helix-20201229003624-c1bf759
           - RedHat.7.Amd64.Open
           - SLES.15.Amd64.Open
-          - (Fedora.34.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20220809205730-e7e8d1c
+          - (Fedora.34.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20220908220224-98605e8
           - (Ubuntu.2204.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-amd64-20220504035722-1b9461f
```

Comment:
Missing
```suggestion
          - (Ubuntu.2204.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-amd64-20220909180913-1b9461f
```

Reply:
I made update to keep it consistent with `main` but the story is ... complicated. 
`Ubuntu 22.04` does not have `OpenSSL 1.1` msquic needs so the official MsQuic packages will not really work out of the box.
We can either force it via tricks to use OpenSSL for other versions or build it from `main` where early support for `OpenSSL 3` exist.

Reply:
BTW `main` also has `Ubuntu.2204.Amd64.Open` e.g. queue without container and that was/will be covered by issue to update Helix queues.

Reply:
OK, in other words the update in main was unnecessary, fair.

Reply:
And you took the change either way here in follow up commit https://github.com/dotnet/runtime/pull/75479/commits/121d1e6a1a22754ed557a583c32eb558263e3d0e